test/integration/protocol_integration_test.cc

```diff
@@ -1578,11 +1578,23 @@ TEST_P(DownstreamProtocolIntegrationTest, LargeRequestHeadersRejected) {
   testLargeRequestHeaders(95, 1, 60, 100);
 }
 
+TEST_P(DownstreamProtocolIntegrationTest, VeryLargeRequestHeadersRejected) {
```

Comment:
I'd think you'd need to exclude this for H3 as well, as we don't have header configuration limits.  We'll see what CI says :-)

Reply:
Added exclude macros